app/utils/security.ts

```diff
@@ -23,3 +25,7 @@ export const hashCode = (str: string): string => {
     }
     return hash.toString();
 };
+
+export const hashServerUrl = (str: string): string => {
```

Comment:
Nit: this isn't a hash. No reason to tie this to server URLs specifically either; there's a good chance we find more use cases for this type of encoding in the future.

A more accurate name would be something like `urlSafeBase64Encode`.